### PR TITLE
Revert homepage to live one

### PR DESCRIPTION
**Pull request by frankieroberto**
This reverts the first page of the prototype to match the [one that’s currently live](https://www.find-postgraduate-teacher-training.service.gov.uk).

The search results page is also updated with a header above the search results showing how they are sorted (if searching by location or 'Across England').

Things that don’t currently work:

* [x] autocomplete for location on first page
* [ ] autocomplete for providers on first page
* [x] filtering on provider name
* [x] filtering on location name
* [ ] changing sort order (alphabetical or reverse alphabetical) doesn't work
* [ ] all courses currently can sponsor visas - (no data on this yet so would need to be faked)
* [ ] degree requirement filter (search results show a random degree class requirement)

Also need to decide how to display locations in search results (current design is from previous iteration and doesn’t match production)

### CODE REVIEW

File: app/views/results.njk [hunk 1]
```diff
@@ -4,9 +4,17 @@
...
+
+  {% if data.initialChoice == "england" %}
+    in <b>England</b>
+  {% elif data.initialChoice == "location" %}
+    in <b>{{ data.locationName }}</b>
+  {% elif data.initialChoice == "provider" %}
+    from <b>{{ data.providerName }}</b>
+  {% endif %}
+  <a href="/" class="govuk-link govuk-!-margin-left-4">Change <span class="govuk-visually-hidden">subject or location</span></a>
 </div>
```
Comment (paulrobertlloyd) [inline app/views/results.njk:18]:
```suggestion
<p class="govuk-body govuk-!-margin-bottom-0">
  <b>{{ selectedSubjects | join("</b> and <b>", "text") }}</b> courses

  {% if data.initialChoice == "england" %}
    in <b>England</b>
  {% elif data.initialChoice == "location" %}
    in <b>{{ data.locationName }}</b>
  {% elif data.initialChoice == "provider" %}
    from <b>{{ data.providerName }}</b>
  {% endif %}
  <a href="/" class="govuk-link govuk-!-margin-left-4">Change <span class="govuk-visually-hidden">subject or location</span></a>
</p>
```

File: app/assets/sass/_accessible-autocomplete.scss [hunk 2]
```diff
@@ -0,0 +1 @@
...order:2px solid #0b0c0c;border-top:0;color:#0b0c0c;margin:0;max-height:342px;overflow-x:hidden;padding:0;width:100%;width:calc(100% - 4px)}.autocomplete__menu--visible{display:block}.autocomplete__menu--hidden{display:none}.autocomplete__menu--overlay{box-shadow:rgba(0,0,0,.256863) 0 2px 6px;left:0;position:absolute;top:100%;z-index:100}.autocomplete__menu--inline{position:relative}.autocomplete__option{border-bottom:solid #b1b4b6;border-width:1px 0;cursor:pointer;display:block;position:relative}.autocomplete__option>*{pointer-events:none}.autocomplete__option:first-of-type{border-top-width:0}.autocomplete__option:last-of-type{border-bottom-width:0}.autocomplete__option--odd{background-color:#fafafa}.autocomplete__option--focused,.autocomplete__option:hover{background-color:#1d70b8;border-color:#1d70b8;color:#fff;outline:0}.autocomplete__option--no-results{background-color:#fafafa;color:#646b6f;cursor:not-allowed}.autocomplete__hint,.autocomplete__input,.autocomplete__option{font-size:16px;line-height:1.25}.autocomplete__hint,.autocomplete__option{padding:5px}@media (min-width:641px){.autocomplete__hint,.autocomplete__input,.autocomplete__option{font-size:19px;line-height:1.31579}}
```
Comment (paulrobertlloyd) [inline app/assets/sass/_accessible-autocomplete.scss:1]: Can we use the same stylesheet (and pretty formatting) as that on the Apply prototype: https://github.com/DFE-Digital/apply-for-teacher-training-prototype/blob/main/app/assets/sass/components/_autocomplete.scss

Comment (frankieroberto) [reply app/assets/sass/_accessible-autocomplete.scss:1]: @paulrobertlloyd yeah, the javascript too I think. Probably ideal to import them direct from the npm module?

Comment (paulrobertlloyd) [reply app/assets/sass/_accessible-autocomplete.scss:1]: Yeah, that would make sense.

**Pull request merged**